doc/src/sgml/rules.sgml

```diff
@@ -1360,7 +1360,7 @@ Time: 16386.245 ms (00:16.386)
 <sect3>
 <title>Subqueries</title>
 <para>
-    Currently, correlated subqueries using <literal>EXISTS</literal> and simple
+    Currently, subqueries using <literal>EXISTS</literal> and simple
      subqueries in <literal>FROM</literal> clause are supported.
```

Comment:
Is this hunk for other fix and not related to this pull request, right?

Reply:
Althouth it has already commited, I don't know why it is included in this pull request.

Reply:
I don't know either, but I wonder maybe you need rebase.

Reply:
Ok I will try to delete this commit by rebase.

Reply:
I deleted it.